samples/dashboards/demo/light-dark-theme/demo.css

```diff
@@ -38,3 +38,8 @@
         fill-opacity: 0.9;
     }
 }
+
+label {
```

Comment:
Instead of overwriting global rules for labels, I would suggest to use `.inputs-wrapper label { }`